recipes/flask-principal/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{%set name = "Flask-Principal" %}
+{%set version = "0.4.0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "f5d6134b5caebfdbb86f32d56d18ee44b080876a27269560a96ea35f75c99453" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+  patches:
+    - 14346d1.patch
+    # Add MANIFEST.in with LICENSE - see https://github.com/mattupstate/flask-principal/blob/master/MANIFEST.in
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - flask
+
+test:
+  imports:
+    - flask_principal
+
+about:
+  home: http://packages.python.org/Flask-Principal/
+  license: MIT
+  license_file: LICENSE
```

Comment:
No worries. Let's just comment this and include a link to PR ( https://github.com/mattupstate/flask-principal/pull/16 ) in the comment.